File: java/src/main/java/dev/spiralmoon/maplestory/api/common/param/LatestApiUpdateTimeOption.java
package dev.spiralmoon.maplestory.api.common.param;

import lombok.AllArgsConstructor;
import lombok.Data;

/**
 * API 서버의 데이터 갱신 시각과 조회 가능한 최근 날짜와 현재 날짜와의 차이
 */
@AllArgsConstructor
@Data
public class LatestApiUpdateTimeOption {
    private int hour;
    private int minute;
    private Integer dateOffset;
}

File: csharp/MapleStory.OpenAPI/Src/Common/Param/LatestApiUpdateTimeOption.cs
namespace MapleStory.OpenAPI.Common.Param
{
    /// <summary>
    /// API 서버의 데이터 갱신 시각과 조회 가능한 최근 날짜와 현재 날짜와의 차이
    /// </summary>
    public struct LatestApiUpdateTimeOption
    {
        public int Hour { get; set; }

        public int Minute { get; set; }

        public int? DateOffset { get; set; }
    }
}
